.github/workflows/integration_tests.yml

```diff
@@ -404,6 +404,14 @@ jobs:
           KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
         run: cd test/go-tests && go test -run SequenceStateIntegrationTest
 
+      - name: Test Sequence Loop
+        id: test_sequence_loop
+        timeout-minutes: 5
+        continue-on-error: true
+        env:
+          KEPTN_ENDPOINT: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
+        run: cd test/go-tests && go test -run SequenceLoopIntegrationTest
+
```

Comment:
we need to add these tests to the test report - see issue https://github.com/keptn/keptn/issues/4018